services/dbaas/users.go

```diff
@@ -16,163 +18,168 @@ import (
 )
 
 func (c *MongoClient) CreateUser(ctx context.Context, clusterId string, user mongo.User) (mongo.User, utils.ApiResponseInfo, error) {
```

Comment:
lint error. clusterId should be clusterID

Reply:
@digna-ionos done, linter errors are fixed.